actions/checkForAndPlaceOrder.ts

```diff
@@ -423,16 +433,15 @@ export const _printUnfilledOrders = (orders: Map<BytesLike, OrderStatus>) => {
  * @param order to be placed on the cow protocol api
  * @param apiUrl rest api url
  */
-async function _placeOrder(
-  orderUid: string,
-  order: any,
-  apiUrl: string,
-  orderRef: string
-): Promise<
-  | Omit<PollResultSuccess, "order" | "signature">
-  | PollResultTryNextBlock
-  | PollResultUnexpectedError
-> {
+async function _placeOrder(params: {
+  orderUid: string;
+  order: any;
```

Comment:
Don't we have a type for the order?

Reply:
we do, but this is from another PR and we will need to code freeze this until @mfw78 merges his waterfall.